docs/navigating-backup.md

```diff
@@ -18,11 +19,71 @@ uses cases, the following settings are available and further documented below.
+
+Say that in your environment you have three regions in which your devices reside: North America, Asia Pacific, and Africa. You have populated these values as `Region` objects in Nautobot, and assigned a `Region` value to each of your devices. You want your backup solution to scale well, so you have a dedicated backup Git repository for each region. Every Nautobot object has a `slug` (URL compatible) name in addition to its human-friendly name; our regions' slugs are `north-america`, `asia-pacific`, and `africa`. To configure the plugin to match devices to the desired Git repository, you must first configure the GraphQL query; a _VERY_ simple one might look like this:
+```
+query ($device_id: ID!) {
+  device(id: $device_id) {
+    config_context
+    hostname: name
+    platform {
+      manufacturer {
+        name
+      }
+      name
+      napalm_driver
+      slug
+    }
+    primary_ip4 {
+      address
+      interface {
+        name
+      }
+      id
+    }
+    site {
+      name
+      region {
+        name
+        slug
+      }
+      slug
+    }
+  }
+}
+```
+
+The query will look at the `Device` ORM object, and return the values from the query as keys under the top-level `obj` key. The `obj` key represents the Device object. With this GraphQL query, we can make a Jinja2 template to translate the returned values into a string. For example, say that you have a device which is in your Sydney, AU office, which is in the `asia-pacific` region in Nautobot. If you made a Jinja2 template based on that, which looked like this:
+
+```
+{{obj.site.region.slug}}
+```
+Then the template would be rendered to the string:
+```
+asia-pacific
+```
+
+When you create backup repositories, pay attention to your naming scheme. You should name each repository in a way that matches the value of whatever parameter from the Device object which you wish to use to sort devices into repositories. So, for our Sydney device above, it would work to name your Asia Pacific repository something "Asia Pacific Device Backups". This would give it a `slug` value of `asia-pacific-device-backups`, and you could use this in a backup repository matching rule with a template like this:
+
+```
+{{obj.site.region.slug}}-device-backups
+```
+
+### Backup Path Template
+
+The `backup_path_template` setting gives you a way to dynamically place each device's configuration file in the repository file structure. This setting uses the GraphQL query configured for the plugin. It works in a similar way to the Backup Repository Matching Rule above. Since the setting uses a GraphQL query, any valid Device model method is available. The plugin renders the values from the query, using Jinja, to the relative path and file name in which to store a given device's configuration inside its backup repository. This may seem complicated, but the equivalent of `obj` by example would be:
```

Comment:
```suggestion
The `backup_path_template` setting gives you a way to dynamically place each device's configuration file in the repository file structure. This setting uses the GraphQL query configured for the plugin. It works in a similar way to the Backup Repository Matching Rule above. Since the setting uses a GraphQL query, any valid Device model method is available. The plugin renders the values from the query, using Jinja2, to the relative path and file name in which to store a given device's configuration inside its backup repository. This may seem complicated, but the equivalent of `obj` by example would be:
```

Reply:
Nit picky for sure but with the addition and cleanup of many of the docs felt it would be a good time to get it cleaned up across the board in the docs to using `jinja2` verbiage.  Other than this I'm good with approving.